kusto/ingest/file_options.go

```diff
@@ -459,3 +459,17 @@ func ClientRequestId(clientRequestId string) FileOption {
 		name:         "ClientRequestId",
 	}
 }
+
+// RawDataSize is the uncompressed data size. Should be used to comunicate the file size to the service for efficient ingestion.
+// Also used by managed client in the decision to use queued ingestion instead of streaming (if > 4mb)
```

Comment:
But, following our discussion, this is not true.
The compressed estimate is the one with the 4MB limit, not the raw one.

Reply:
That was before this decision and is currently true - we dont have any estimation in the go client so this will change will have to be in a new version together with all the other sdks

Reply:
After discussion - Done in this version